vulture_whitelist.py

```
@@ -0,0 +1,407 @@
+#
+# Most of these are dataclass fields / enum members that mirror GraphQL
+# response shapes; they're consumed by the separate home-assistant-frank_energie
+# integration repo, which vulture can't see. A few are pytest hooks/fixtures
+# vulture can't detect are auto-invoked, and a few (month_insights,
+# contract_price_resolution_state, smart_hvac_status, format_delivery_site_as_dict,
+# for_current_quarter_hour, differenceUntilLastMeterReadingDateAvg, length) are
+# confirmed real usages in that external repo.
+#
+# Regenerate with: poetry run vulture --min-confidence 60 --make-whitelist python_frank_energie tests
+
+ELECTRICITY  # unused variable (python_frank_energie/domain.py:14)
```

Flake8 (7.3.0)

[error] undefined name 'ELECTRICITY'

(F821)